Sources/CorePayments/Networking/APIClient.swift

```diff
@@ -70,4 +85,16 @@ public class APIClient {
         
         return url
     }
+    
+    private func constructGraphQLURL(queryName: String? = nil) throws -> URL {
+        guard let queryName else {
+            return coreConfig.environment.graphQLURL
+        }
+        
+        if let url = URL(string: coreConfig.environment.graphQLURL.absoluteString + "?" + queryName) {
```

Comment:
Above in the `constructRestURL` method we use a guard then return the URL. Should we use that here instead to keep the methods consistent vs an `if let`?

Reply:
Done in 2d5a36a4e3bb0ce8031c7a298cab85d5f73e8bf7!